modules/juce_audio_devices/native/juce_mac_CoreAudio.cpp

```diff
@@ -2152,7 +2187,13 @@ class CoreAudioIODeviceType  : public AudioIODeviceType,
         auto combinedName = outputDeviceName.isEmpty() ? inputDeviceName
                                                        : outputDeviceName;
 
-        if (inputDeviceID == outputDeviceID)
+        // Newer Apple platforms can create aggregate audio devices
+        bool canCreateAppleAggregateDevice = [] {

```

Comment:
Why is this in a lambda?

Reply:
If I use `@available` and not `if (@available...` Xcode warns that it's not actually guarding availability. Given that, this seemed the right way to me